components/bl-granim-component/README.md

```diff
@@ -0,0 +1,76 @@
+|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|------------------------|--------------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| States <br> `states`                                     | *JSON*                                                                                                                                                                                                                    | `{}`          | States Logic           | YES          | NO         | This is a handler for the logic to determine the object containing all the states. Signature of states: {}. Signature of state: { gradients: [...[...String]] \|\| [...{color: String, pos: Number}], transitionSpeed: Number, loop: Boolean}. Watch [Codeless Examples](#Examples). |
+| Direction <br> `direction`                               | *Select* <br> [`left-right`, `diagonal`, `top-bottom`, `radial`, `custom`]                                                                                                                                                | `left-right`  |                        | NO           | YES        | Controls the orientation of the gradient.                                                                                                                                                                                                                                            |
+| Custom Direction <br> `customDirection`                  | *JSON*                                                                                                                                                                                                                    | `{}`          | Custom Direction Logic | YES          | NO         | This is a handler for the logic to determine the custom direction of the gradient. Signature of custom direction with pixels or percentage values: {x0: String, y0: String, x1: String, y1: String}. Watch [Codeless Examples](#Examples).                                           |
```

Comment:
`x0` - is string? JSON pretty well works with numbers, we can't change it?

Reply:
These parameters can be in pixels or percentage, so we can't limit them to Number type.